src/go/k8s/apis/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -28,8 +28,8 @@ type RedpandaClusterSpec struct {
 
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 
-	LicenseKey       *string           `json:"license_key,omitempty"`
-	LicenseSecretRef *LicenseSecretRef `json:"license_secret_ref,omitempty"`
+	// +kubebuilder:pruning:PreserveUnknownFields
+	Enterprise *runtime.RawExtension `json:"enterprise,omitempty"`
```

Comment:
Leave this out for now, ill update tomorrow if you need it asap. I do not think we want this to be RawExtension.